guides/manual-installation.md

```diff
@@ -0,0 +1,505 @@
+CREATE DATABASE trento_event_store;
+```
+
+**Step 3:** Create the users:
+
+```sql
+CREATE USER wanda_user WITH PASSWORD 'wanda-password';
+CREATE USER trento_user WITH PASSWORD 'web-password';
+```
+
+**Step 4:** Grant required privileges to the users and close the connection:
+
+```sql
+\c wanda
+GRANT ALL ON SCHEMA public TO wanda_user;
+\c trento
+GRANT ALL ON SCHEMA public TO trento_user;
+\c trento_event_store;
+GRANT ALL ON SCHEMA public TO trento_user;
+\q
+```
+
+**Step 5:** Allow the docker containers to connect to their respective databases by adding the following in `/var/lib/pgsql/data/pg_hba.conf`:
+
+```bash
+host    wanda           wanda_user      0.0.0.0/0               md5
+host    trento          trento_user     0.0.0.0/0               md5
+host    trento_event_store      trento_user     0.0.0.0/0       md5
+```
+
+**Step 6:** Allow PostgreSQL to bind on all interfaces `/var/lib/pgsql/data/postgresql.conf` by changing the following line:
+
+> Note: for a more granular approach, you can specify the IP address of the docker host instead: `172.17.0.1/32` is a common default for docker.
+
+```bash
+listen_addresses = '*'
+```
+
+**Step 7:** Restart postgres to apply the changes:
+
+```bash
+systemctl restart postgresql
+```
+
+### Install RabbitMQ
+
+```bash
+zypper install rabbitmq-server
+```
+
+As the agent needs to reach RabbitMQ, allow connections from external hosts.
+Modify `/etc/rabbitmq/rabbitmq.conf` and ensure the following lines are present:
+
+```bash
+listeners.tcp.default = 5672
+```
+
+Add an exception on firewalld:
+
+```bash
+firewall-cmd --zone=public --add-port=5672/tcp --permanent;
+firewall-cmd --reload
+```
+
+As the agent needs to reach RabbitMQ, allow connections from external hosts:
+
+```bash
+systemctl enable --now rabbitmq-server
+```
+
+### Configure RabbitMQ
+
+> Note: In order to configurate RabbitMQ for a production system, follow the official suggestions [RabbitMQ guide](https://www.rabbitmq.com/production-checklist.html)
+
+Create a new RabbitMQ user and change the following credentials:
+
+- rabbitmq_user
+- rabbitmq_user_password
+- vhost
+
+```bash
+rabbitmqctl add_user trento_user trento_user_password
+```
+
+Set role for new user:
+
+```bash
+rabbitmqctl set_user_tags trento_user administrator
+```
+
+Verify that the user and the role are correct:
+
+```bash
+rabbitmqctl list_users
+```
+
+Create a virtual host:
+
+```bash
+rabbitmqctl add_vhost vhost
+```
+
+Set permissions for the user on the virtual host:
+
+```bash
+rabbitmqctl set_permissions -p vhost trento_user ".*" ".*" ".*"
+```
+
+### Install container runtime
+
+First we need to enable the containers module:
+
+```bash
+SUSEConnect --product sle-module-containers/15.5/x86_64
+```
+
+Then we can install docker:
+
+```bash
+zypper install docker
+```
+
+Enable and start Docker:
+
+```bash
+systemctl enable --now docker
+```
+
+### Deploy trento-wanda and trento-web components on docker:
+
+#### Create the secrets
```

Comment:
Please don't refer as them by "secrets" this could be misleading in a docker context, just env variables